www/_template/reference/configuration.md

```diff
@@ -5,110 +5,325 @@ description: The Snowpack configuration API reference.
+    "public": {url: "/", static: true, resolve: false}
+  }
+}
+```
+
+- `mount.url` | `string` | _required_ : The URL to mount to, matching the string in the simple form above.
+- `mount.static` | `boolean` | _optional_ | **Default**: `false` : If true, don't build files in this directory. Copy and serve them directly from disk to the browser.
+<!-- TODO: does this still exist?
+- `staticHtml` _optional, **Default**: false_: If true, don't build HTML (`.html`) files in this directory. This special option exists because HTML files are almost always built to support HMR and the popular pattern of keeping HTML files in a `public/` directory that's otherwise full of static. -->
+- `mount.resolve` | `boolean` | _optional_ | **Default**: `true`: If false, don't resolve JS & CSS imports in your JS, CSS, and HTML files. Instead send every import to the browser, as written.
+-
+
+## config.alias
+
+**Type**: `object` (package: package or path)
+
+Configure import aliases for directories and packages.
+
+> Note: In an older version of Snowpack, all mounted directories were also available as aliases by **Default**. As of Snowpack 2.7, this is no longer the case and no aliases are defined by **Default**.
+
+```js
+// snowpack.config.js
+// Example: alias types
+{
+  alias: {
+    // Type 1: Package Import Alias
+    "lodash": "lodash-es",
+    "react": "preact/compat",
+    // Type 2: Local Directory Import Alias (relative to cwd)
+    "components": "./src/components"
+    "@app": "./src"
+  }
+}
+```
+
+## config.plugins
+
+**Type**: `array` containing pluginName `string` or an array [`pluginName`, {`pluginOptions`}
+
+Enable Snowpack plugins and their options
+
+```js
+// snowpack-config.js
+// Example: enable plugins both simple and expanded
+{
+  plugins: ['plugin-1', ['plugin-2', { 'plugin-option': false }]];
+}
+```
+
+## config.devOptions
+
+**Type**: `object` (option name: value)
+
+Configure the Snowpack dev server.
+
+### devOptions.secure
+
+**Type**: `boolean`
+
+**Default**: `false`
+
+Toggles whether Snowpack dev server should use HTTPS with HTTP2 enabled.
+
+### devOptions.hostname
+
+**Type**: `string`
+
+**Default**: `localhost`
+
+The hostname the dev server runs on.
+
+TODO: originally "The hostname where the browser tab will be open." is this more correct? Also why isn't it in the devoptions object in config.ts?
+
+### devOptions.port
+
+**Type**: `number`
+
+**Default**: `8080`
+
+The port the dev server runs on.
+
+### devOptions.fallback
+
+**Type**: `string`
+
+**Default**: `"index.html"`
+
+The HTML file to serve for non-resource routes.
+
+When using the Single-Page Application (SPA) pattern, this is the HTML "shell" file that gets served for every (non-resource) user route.
+
+⚠️ Make sure that you configure your production servers to serve this.
+
+### devOptions.bundle
+
+**Type**: `boolean`
+
+**Default**: TODO
+
+TODO: This is still in config.ts but seems deprecated
```

Comment:
@FredKSchott also wasn't sure about `devOptions.bundle`- what it does or whether it's just deprecated

Reply:
deprecated!